packages/lib-utils/src/components/status/StatusBox.tsx

```diff
@@ -3,11 +3,14 @@ import * as _ from 'lodash-es';
 import * as React from 'react';
 
 export type LoadError = {
+  /* Error message. */
```

Comment:
Nit: please use `/**` comment blocks, since `/*` ones won't appear in JSdoc.

Reply:
Missed that, thank you @vojtechszocs